src/player/PlayerComponent.cpp

```diff
@@ -630,12 +636,13 @@ void PlayerComponent::handleMpvEvent(mpv_event *event)
       }
       // Start "on_preloaded" hook.
       // Used initialize stream selections and to probe codecs.
-      if (!strcmp(msg->args[1], "2"))
+      if (!strcmp(hook->name, "on_preloaded"))
```

Comment:
Not this PRs fault but should use `== false`.

Reply:
Is that the standard in PMP? We switched to `!` in PMS a while back.

Reply:
No we use `!` everywhere else

Reply:
The PMS spec still says `== false`, it does look like PMP is always using `!` so ignore me.